solr_conf/conf/solrconfig.xml

```diff
@@ -738,8 +738,8 @@
         type_s
         type_t
         collection_ss
-        shelfmark_t
-        shelfmark_ss
+        shelfmark_t^80
```

Comment:
Boosts look good. As a developer, do I need to copy this file again to my solr home directory and/or run any solr commands for the changes to go into effect?

Reply:
yes, you'll have to update the configset or core config with the changed files and reload the solr core.

I was thinking to warn everyone about the change in slack once this gets merged in; but we might want to think about a better approach for that, since we will likely be making more changes to the solr config